packages/theme/pages/MyAccount/MyProfile.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <SfTabs :open-tab="1">
+    <!-- Personal data update -->
+    <SfTab title="Personal data">
+      <p class="message">
+        {{ $t('Feel free to edit') }}
+      </p>
+
+      <ProfileUpdateForm @submit="updatePersonalData" />
+
+      <p class="notice">
+        {{ $t('Use your personal data') }}
+        <a href="">{{ $t('Privacy Policy') }}</a>
+      </p>
+    </SfTab>
+    <!-- Password reset -->
+    <SfTab title="Password change">
+
+      <p class="message">
+        {{ $t('Change password your account') }}:<br />
+      </p>
+      <PasswordResetForm @submit="updatePassword" />
```

Comment:
Is there any reason to propagate the submit event to the page component? It seems like it could've been handled inside PasswordResetForm component (you can also use the useUser composable inside it).
The same for ProfileUpdateForm

Reply:
It is done that way in the vsf "base" so I just let it be as it was, but You've got a point

Reply:
Ah, right. It feels a bit overcomplicated, but maybe there was a reason for that. Are we able to simplify it in our implementation easily?